TestCntlrStub/src/ts_utls.c

```diff
@@ -1765,11 +1765,11 @@ int tsSendResetReq(unsigned int resetType, int numOfUes, unsigned char *ueLst)
       ((ResetReq*)msgptr)->cause.causeVal = 1;
 
       ((ResetReq*)msgptr)->r.partialRst.numOfConn = numOfUes;
-      ((ResetReq*)msgptr)->r.partialRst.ueIdLst = malloc(numOfUes);
-      memcpy(((ResetReq*)msgptr)->r.partialRst.ueIdLst, ueLst, numOfUes);
+      ((ResetReq*)msgptr)->r.partialRst.ueS1apIdPairList = malloc(numOfUes);
+      memcpy(((ResetReq*)msgptr)->r.partialRst.ueS1apIdPairList, ueLst, numOfUes);
```

Comment:
Similar to the above comment, this memcpy here should be updated as well. Moreover, the "ueLst" variable needs to be updated  for this function with the new structure of "ueS1apIdPairList".